src/lib/errorHandler.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * This code snippet handles any un-handled errors from occuring in the consumer iframe.
+ * This is a global event handler that will capture all the error messages and log them to the console.
+ * Once the error is logged to the console this onerror event handler will suppress the script error from surfacing on the UI,
+ * avoiding an interruption to the user experience by not displaying a script error.
+ *
+ * Reason for this change : In some cases when the xfc library is used in a nested iframe architecture, it results in a situation
+ * where if the outer iframe is cleared quick enough to force a re-load then postMessages from the nested iFrame result in script
+ * errors during reload.
+ */
+
+/* eslint no-console: 0 */
+
+if (window && !window.onerror) {
```

Comment:
Is there an example of errors being thrown? I am curious replicate this in our consuming app.

Reply:
I have not been able to recreate the error using a single instance of xfc being used. This happens in the nested setting when you have xfc being consumed at the top level as well as its used to frame another 3rd party application inside the nested iFrame.  

The error thrown would be incase of the minified JS: 
Object expected

Reply:
@foxannefoxanne I created a project to simulate some of how SMART uses xfc here  : https://github.com/shriniketsarkar/xfc/tree/simulate_multiple_frames_sortable_with_error_log

Once you clone the project and run it using ```npm run dev``` you can navigate to : http://0.0.0.0:8080/example/embedded_app_lifecycle/2_c_m_r_index.html and click the ToggleFrames button to generate the error log. I am simply generating an error by accessing a function that does not exist to simulate the log. 

In this example the SMART app is just a text blob but the use of xfc is similar to how SMART uses it. 
<img width="1675" alt="Screen Shot 2021-06-30 at 9 15 58 AM" src="https://user-images.githubusercontent.com/45810410/123977041-47e48080-d984-11eb-953f-2116913f0c7e.png">